src/App.js

```diff
@@ -1,29 +1,18 @@
-import logo from './logo.svg';
-import './App.css';
-
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+import React from 'react';
+import Calculator from './components/Calculator';
+import './index.css';
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- [ ] We are not permitted to change Linters's rules. Let's fix the issue.

Reply:
Can't fix them without disabling that. It works this way

Reply:
@Christabelle143 I will advise you to extend React pure component. This [link](https://www.geeksforgeeks.org/reactjs-pure-components/) will surely help.

Reply:
This suggestion are optional right??

Reply:
Not optional.

Reply:
To get rid of the Linters issues, your `Calculator` component should look like this:

import React from ‘react’;
  
```.js
export default class Calculator extends React.PureComponent{
   render(){
      return <h1>blablabla</h1>;
   }
}
```

Kindly do the same with your App component.

Reply:
Thanks it worked.

Reply:
Wish you all the best!